### DISCUSSION TITLE
No sounds when returning to Arcade games...

### DISCUSSION THREAD

**Discussion by boxallw** [OnionUI/Onion#525] [Q&A]
When the game auto saves state, when I return to the game there is no sound. I must manually enter the retroarch menu and restart the game. Any ideas how to fix?

**Comment by Olywa** [comment]
This is a known issue with some of the RetroArch cores. There is no fix for it yet.

**Reply by boxallw** [reply]
Thanks. A quick restart fixes things at least.
